<?xml version="1.0"?>

<launch>
    <arg name="hz" default="20"/>
    <arg name="robot_param" default="$(find dwa_planner)/config/robot_param.yaml"/>
    <arg name="dwa_param" default="$(find dwa_planner)/config/dwa_param.yaml"/>

    <node pkg="dwa_planner" type="dwa_planner" name="dwa_planner" ns="local_planner">
        <rosparam command="load" file="$(arg robot_param)"/>
        <rosparam command="load" file="$(arg dwa_param)"/>
        <param name="HZ" value="$(arg hz)"/>
    </node>
</launch>
